Content.Shared/Chemistry/Components/HyposprayComponent.cs

```diff
@@ -8,6 +8,14 @@ namespace Content.Shared.Chemistry.Components;
 [RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
 public sealed partial class HyposprayComponent : Component
 {
+    [DataField]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MaxPressure = float.MaxValue;
+
+    [DataField]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float InjectTime = 2f;
```

Comment:
```suggestion
    [DataField]
    public float MaxPressure = float.MaxValue;

    [DataField]
    public float InjectTime = 2f;
```
DataField already contains ViewVariables(VVAccess.ReadWrite) so including both isn't necessary.